test/auth_aws/test_auth_aws.py

```diff
@@ -53,6 +56,57 @@ def test_connect_uri(self):
         with MongoClient(self.uri) as client:
             client.get_database().test.find_one()
 
+    def setup_cache(self):
```

Comment:
Should we skip these tests on pymongo_auth_aws <=1.0.2?

Reply:
Added a guard to check for the attribute